extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/SparkSessionProvider.scala

```diff
@@ -46,6 +48,7 @@ trait SparkSessionProvider {
       .config(
         "spark.sql.warehouse.dir",
         Files.createTempDirectory("spark-warehouse").toString)
+      .config("spark.sql.extensions", sqlExtensions)
```

Comment:
why do we need this?

Reply:
It's for adding Iceberg sql extension for testing rewritten commands  in iceberg catalog.
"org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions"